src/controlplane/controlplane_test.go

```diff
@@ -101,8 +112,10 @@ func Test_Scraper_Autodiscover_all_cp_components(t *testing.T) {
 				t.Fatalf("running scraper: %v", err)
 			}
 
+			// Include specific specific exclusions for this version.
+			versionAsserter := asserter.Excluding(apiServer123DeprecationExclusion(version))
```

Comment:
Something like (string comparison abused for brevity):
```suggestion
			if version >= testutil.Testdata123 {
				asserter = asserter.Excluding(apiServer123DeprecationExclusions())
			}
```